aptos-move/aptos-debugger/src/aptos_debugger.rs

```diff
@@ -57,12 +57,24 @@ impl AptosDebugger {
         &self,
         version: Version,
         txns: Vec<Transaction>,
+        repeat_execution_times: u64,
     ) -> Result<Vec<TransactionOutput>> {
         let sig_verified_txns: Vec<SignatureVerifiedTransaction> =
             txns.into_iter().map(|x| x.into()).collect::<Vec<_>>();
         let state_view = DebuggerStateView::new(self.debugger.clone(), version);
-        AptosVM::execute_block_no_limit(&sig_verified_txns, &state_view)
-            .map_err(|err| format_err!("Unexpected VM Error: {:?}", err))
+        for i in 0..repeat_execution_times {
+            let result = AptosVM::execute_block_no_limit(&sig_verified_txns, &state_view)
```

Comment:
Shall we also (maybe add an option to) check if results are the same for every run?

Reply:
@grao1991 added the comparison